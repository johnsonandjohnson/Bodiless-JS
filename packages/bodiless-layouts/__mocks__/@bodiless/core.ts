const editContext = {
  activate: jest.fn(),
  isEdit: true,
};

export const useEditContext = jest.fn(() => editContext);

const activateOnEffect = {
  setId: jest.fn(),
};
export const useActivateOnEffect = () => activateOnEffect;

export const useNode = jest.fn();

const contextMenuFormInner = jest.fn();
export const contextMenuForm = jest.fn(() => contextMenuFormInner);
