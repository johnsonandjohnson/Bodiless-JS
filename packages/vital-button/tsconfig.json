{
  "extends": "../../tsconfig.json",
  "compilerOptions": {
    "rootDir": "src",
    "outDir": "lib",
    "baseUrl": ".",
    "preserveSymlinks": false,
  },
  "include": [
    "src",
    "../../@types/*",
    "src/components/ButtonTokens/index.ts"
  ],
  "exclude": [
    "lib",
    "node_modules",
    "__tests__/**/*.*",
    "src/**/__tests__/**/*.*"
  ]
}