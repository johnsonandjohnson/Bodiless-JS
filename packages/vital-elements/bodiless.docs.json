{
  "VitalDesignSystem": {
    "README.md": "./doc/VitalDesignSystem.md",
    "GettingStarted": {
      "README.md": "./doc/GettingStarted/README.md",
      "CoreConcepts.md": "./doc/GettingStarted/CoreConcepts.md",
      "ComponentsAndTokens.md": "./doc/GettingStarted/ComponentsAndTokens.md",
      "CompositionFromWithout.md": "./doc/GettingStarted/ComposingFromOutside.md",
      "ReachingInside.md": "./doc/GettingStarted/ReachingInside.md",
      "AnatomyOfA_Token.md": "./doc/GettingStarted/AnatomyOfAToken.md",
      "FunctionalClasses.md": "./doc/GettingStarted/FunctionalClasses.md",
      "WorkingWithContent.md": "./doc/GettingStarted/WorkingWithContent.md",
      "CleanComponents.md": "./doc/GettingStarted/CleanComponents.md",
      "YourFirstToken.md": "./doc/GettingStarted/YourFirstToken.md",
      "WhyVital.md": "./doc/GettingStarted/WhyVital.md"
    },
    "Components": {
      "README.md": "./doc/Components.md",
      "VitalElements": {
        "README.md": "./doc/VitalElements.md",
        "ComponentTemplate.md": "./doc/ComponentTemplate.md",
        "ExtendingAndComposingTokens.md": "./doc/ExtendingAndComposing.md",
        "SiteTypography.md": "./doc/SiteTypography.md",
        "Shadow.md": "./doc/Shadow.md",
        "StaticReplacement.md": "./doc/StaticReplacement.md",
        "Tokens": {
          "README.md": "./doc/Tokens.md",
          "TokenDomains.md": "./doc/TokenDomains.md"
        }
      },
      "assets": {
        "PXtoREMTemplate.xlsx": "./doc/assets/PXtoREMTemplate.xlsx"
      }
    }
  }
}
