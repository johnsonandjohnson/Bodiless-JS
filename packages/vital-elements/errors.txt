Errors for Brand Tokens/White Label/Semantic/Color/Background/Section/Light Theme/Secondary
 . Attempt to resolve alias to undefined
Errors for Brand Tokens/White Label/Semantic/Color/Background/Section/Light Theme/Tertiary
 . Attempt to resolve alias to undefined
Errors for Brand Tokens/White Label/Components/Checkbox/Background/Dark Theme/Hover
 . Color value "#FFFFFF" is not an alias
Errors for Brand Tokens/White Label/Components/Checkbox/Border/Dark Theme/Disabled
 . Color value "#FFFFFF" is not an alias
Errors for Brand Tokens/White Label/Components/Checkbox/Border/Dark Theme/Checked
 . Color value "#FFFFFF" is not an alias
Errors for Brand Tokens/White Label/Components/Checkbox/Border/Dark Theme/Hover
 . Color value "#FFFFFF" is not an alias
Errors for Brand Tokens/White Label/Components/Checkbox/Border/Dark Theme/Idle
 . Color value "#FFFFFF" is not an alias
Errors for Brand Tokens/White Label/Components/Checkbox/Background/Light Theme/Idle
 . Component color alias "(Color/--NO VALUE--" is not semantic
Errors for Brand Tokens/White Label/Components/Checkbox/Background/Dark Theme/Disabled
 . Color value "#FFFFFF" is not an alias
Errors for Brand Tokens/White Label/Components/Checkbox/Background/Dark Theme/Idle
 . Color value "#FFFFFF" is not an alias
Errors for Brand Tokens/White Label/Components/Checkbox/Background/Dark Theme/Checked
 . Color value "#FFFFFF" is not an alias
Errors for Brand Tokens/White Label/Components/Form Field/Text/Caption
 . Component color alias "(Typography/Body/Small" is not semantic
Errors for Brand Tokens/White Label/Components/Form Field/Border/Error
 . Component color alias "(Color/Signal/Error/Error" is not semantic
Errors for Brand Tokens/White Label/Components/Form Field/Text/Error
 . Component color alias "(Color/Signal/Error/Error" is not semantic
Errors for Brand Tokens/White Label/Components/Form Field/Border/Success
 . Component color alias "(Color/Signal/Success/Success" is not semantic
Errors for Brand Tokens/White Label/Components/Form Field/Text/Input
 . Component color alias "(Typography/Body/Regular" is not semantic
Errors for Brand Tokens/White Label/Components/Form Field/Text/Label
 . Component color alias "(Typography/Eyebrow/Eyebrow" is not semantic
Errors for Brand Tokens/White Label/Components/Navigation/Link
 . Color value "#FFFFFF" is not an alias
Errors for Brand Tokens/White Label/Components/Navigation/Primary/Primary Link/Border/Hover
 . Component color alias "(Semantic/Color/Interactive/Default" is not semantic
Errors for Brand Tokens/White Label/Components/Navigation/Primary/Primary Link/Border/Active
 . Component color alias "(Semantic/Color/Interactive/Hover" is not semantic
Errors for Brand Tokens/White Label/Components/Category Card/Padding/Bottom
 . Spacing value "0" is not an alias
Errors for Brand Tokens/White Label/Components/Category Card/Padding/Top
 . Spacing value "0" is not an alias
Errors for Brand Tokens/White Label/Components/Category Card/Background/Light Theme/Background
 . Component color alias "(Color/--NO VALUE--" is not semantic
Errors for Brand Tokens/White Label/Components/Category Card/Background/Dark Theme/Background
 . Component color alias "(Color/--NO VALUE--" is not semantic
Errors for Brand Tokens/White Label/Components/Accordion/Dropdown/Border/Light Theme/Border
 . Component color alias "(Color/--NO VALUE--" is not semantic
Errors for Brand Tokens/White Label/Components/Breadcrumb/Background/Light Theme/Color
 . Component color alias "(Semantic/Color/Background/Page/Primary" is not semantic
Errors for Brand Tokens/White Label/Components/Breadcrumb/Background/Dark Theme/Color
 . Component color alias "(Semantic/Color/Background/Page/Primary" is not semantic
Errors for Brand Tokens/White Label/Components/Article Card/Border/Dark Theme/Background
 . Component color alias "(Color/--NO VALUE--" is not semantic
Errors for Brand Tokens/White Label/Components/Article Card/Border/Light Theme/Background
 . Component color alias "(Color/--NO VALUE--" is not semantic
