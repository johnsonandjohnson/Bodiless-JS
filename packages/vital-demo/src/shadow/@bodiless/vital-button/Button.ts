/**
 * Copyright Â© 2022 Johnson & Johnson
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 * http://www.apache.org/licenses/LICENSE-2.0
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

import { asButtonToken } from '@bodiless/vital-button';
import { vitalButtonBase } from '@bodiless/vital-button/lib/base';
import { addProps } from '@bodiless/fclasses';

const Plain = asButtonToken(vitalButtonBase.Plain, {
  Behavior: {
    Wrapper: addProps({ 'data-shadowed-by': 'vital-demo:DefaultButton' }),
  },
});

const WithPrimaryStyle = asButtonToken(
  vitalButtonBase.Plain,
  vitalButtonBase.WithPrimaryStyle, {
    Behavior: {
      Wrapper: addProps({ 'data-shadowed-by': 'vital-demo:PrimaryButton' }),
    },
  }
);

const WithSecondaryStyle = asButtonToken(
  vitalButtonBase.Plain,
  vitalButtonBase.WithSecondaryStyle, {
    Behavior: {
      Wrapper: addProps({ 'data-shadowed-by': 'vital-demo:SecondaryButton' }),
    },
  }
);

const WithTertiaryStyle = asButtonToken(
  vitalButtonBase.Plain,
  vitalButtonBase.WithTertiaryStyle, {
    Behavior: {
      Wrapper: addProps({ 'data-shadowed-by': 'vital-demo:TertiaryButton' }),
    },
  }
);

export default {
  ...vitalButtonBase,
  Plain,
  WithPrimaryStyle,
  WithSecondaryStyle,
  WithTertiaryStyle,
};
