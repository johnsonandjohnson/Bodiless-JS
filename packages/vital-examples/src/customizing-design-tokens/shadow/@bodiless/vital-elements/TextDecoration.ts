import { shadow } from '@bodiless/vital-elements';
import { exampleTextDecoration } from '../../../components/TextDecoration';

export default shadow(exampleTextDecoration, 'Example:TextDecoration');
