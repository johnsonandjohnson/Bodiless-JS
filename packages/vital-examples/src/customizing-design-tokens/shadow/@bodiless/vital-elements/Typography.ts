import { shadow } from '@bodiless/vital-elements';
import { exampleTypography } from '../../../components/Typography';

export default shadow(exampleTypography, 'Example:Typography');
