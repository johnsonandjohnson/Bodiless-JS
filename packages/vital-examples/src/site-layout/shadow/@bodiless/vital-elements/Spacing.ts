import { shadow } from '@bodiless/vital-elements';
import { exampleSpacing } from '../../..';

export default shadow(exampleSpacing, 'Example:Spacing');
