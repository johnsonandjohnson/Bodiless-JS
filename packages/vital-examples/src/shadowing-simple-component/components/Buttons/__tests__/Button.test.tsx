/* eslint-disable jest/valid-describe */
import { ButtonClean } from '@bodiless/vital-button';
import { testTokens } from '@bodiless/vital-elements';
import { exampleButtons } from '..';

describe('Button', testTokens(ButtonClean, exampleButtons));
