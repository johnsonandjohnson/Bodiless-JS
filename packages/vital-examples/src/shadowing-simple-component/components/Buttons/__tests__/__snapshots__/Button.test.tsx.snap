// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Matches snapshot for Plain 1`] = `
<a
  className="rounded-8px px-24px py-16px group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      className="text-14px lg:text-16px leading-6 font-bold font-2"
      dangerouslySetInnerHTML={
        Object {
          "__html": "BUTTON",
        }
      }
    />
  </span>
</a>
`;

exports[`Button Matches snapshot for PrimaryDisabled 1`] = `
<button
  className="bg-kenvue-neutrals-grey"
  disabled="true"
>
  <span
    className="text-kenvue-neutrals-white"
  />
</button>
`;

exports[`Button Matches snapshot for SecondaryDisabled 1`] = `
<button
  className="border-kenvue-neutrals-grey"
  disabled="true"
>
  <span
    className="text-kenvue-neutrals-grey"
  />
</button>
`;

exports[`Button Matches snapshot for TertiaryDisabled 1`] = `
<button
  disabled="true"
>
  <span
    className="text-kenvue-neutrals-grey"
  />
</button>
`;

exports[`Button Matches snapshot for WhereToBuy 1`] = `
<a
  className="rounded text-m-base lg:text-base uppercase font-bold text-vital-primary-footer-copy bg-vital-primary-interactive disabled:bg-vital-primary-button-disabled mx-auto p-3 w-full flex justify-center items-center max-w-64 h-12 lg:w-full"
  href="/where-to-buy"
>
  <svg
    className="w-6 h-6 mr-3 xl:mr-0 2xl:mr-3"
    height="25"
    viewBox="0 0 27.27 25"
    width="27.27"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="CART ICON"
      id="CART_ICON"
      transform="translate(1 1)"
    >
      <path
        d="M-2020.75,2766.846h4.712l3.426,15.993h14.314l2.818-11h-19.487"
        data-name="Path 5710"
        fill="none"
        id="Path_5710"
        stroke="#fff"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="2"
        transform="translate(2020.75 -2766.846)"
      />
      <circle
        cx="2.404"
        cy="2.404"
        data-name="Ellipse 1822"
        fill="#fff"
        id="Ellipse_1822"
        r="2.404"
        transform="translate(8.019 19.192)"
      />
      <circle
        cx="2.404"
        cy="2.404"
        data-name="Ellipse 1823"
        fill="#fff"
        id="Ellipse_1823"
        r="2.404"
        transform="translate(17.657 19.192)"
      />
    </g>
  </svg>
  <span
    className="leading xl:hidden 2xl:block"
  >
    Where to Buy
  </span>
</a>
`;

exports[`Button Matches snapshot for WhereToBuyWithoutIcon 1`] = `
<a
  className="rounded text-m-base lg:text-base uppercase font-bold text-vital-primary-footer-copy bg-vital-primary-interactive disabled:bg-vital-primary-button-disabled mx-auto p-3 w-full flex justify-center items-center max-w-64 h-12 lg:w-full"
  href="/where-to-buy"
>
  <span
    className="leading xl:hidden 2xl:block"
  >
    Where to Buy
  </span>
</a>
`;

exports[`Button Matches snapshot for WithBigButton 1`] = `
<a
  className="rounded-8px px-12 py-6 group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      className="text-14px lg:text-16px leading-6 font-bold font-2"
      dangerouslySetInnerHTML={
        Object {
          "__html": "BUTTON",
        }
      }
    />
  </span>
</a>
`;

exports[`Button Matches snapshot for WithDisabled 1`] = `
<button
  disabled="true"
>
  <span />
</button>
`;

exports[`Button Matches snapshot for WithPrimary 1`] = `
<a
  className="rounded hover:bg-vital-primary-interactive transition-colors duration-400 text-m-base lg:text-base uppercase font-bold text-white bg-vital-primary-brand"
>
  <span
    className="text-kenvue-neutrals-white"
  />
</a>
`;

exports[`Button Matches snapshot for WithPrimaryStyle 1`] = `
<a
  className="active:bg-kenvue-green-shade-60 focus:bg-kenvue-green-shade-80 hover:bg-kenvue-green-shade-80 bg-kenvue-green-green focus:drop-shadow-button focus:outline-none active:drop-shadow-none"
>
  <span
    className="text-kenvue-neutrals-white"
  />
</a>
`;

exports[`Button Matches snapshot for WithSecondaryStyle 1`] = `
<a
  className="active:border-kenvue-green-shade-60 focus:border-kenvue-green-shade-80 hover:border-kenvue-green-shade-80 border-kenvue-green-green border-solid border-1px focus:shadow-button focus:outline-none active:shadow-none"
>
  <span
    className="group-active:text-kenvue-green-shade-60 focus:text-kenvue-green-shade-80 group-hover:text-kenvue-green-shade-80 text-kenvue-green-green"
  />
</a>
`;

exports[`Button Matches snapshot for WithTertiaryStyle 1`] = `
<a
  className="focus:drop-shadow-button focus:outline-none active:drop-shadow-none"
>
  <span
    className="active:text-kenvue-green-shade-60 focus:text-kenvue-green-shade-80 hover:text-kenvue-green-shade-80 text-kenvue-green-green"
  />
</a>
`;

exports[`Buttons Matches snapshot for Default 1`] = `
<a
  className="hover:opacity-80 leading-tight uppercase focus:outline-none focus:ring-0 rounded transition duration-150 ease-in-out px-6 py-3.5 flex group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Link",
        }
      }
    />
  </span>
</a>
`;

exports[`Buttons Matches snapshot for Primary 1`] = `
<a
  className="text-white bg-vital-primary-interactive hover:opacity-80 leading-tight uppercase focus:outline-none focus:ring-0 rounded transition duration-150 ease-in-out px-6 py-3.5 flex group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Link",
        }
      }
    />
  </span>
</a>
`;

exports[`Buttons Matches snapshot for PrimarySelected 1`] = `
<a
  className="text-white bg-vital-secondary-button-selected hover:opacity-80 leading-tight uppercase focus:outline-none focus:ring-0 rounded transition duration-150 ease-in-out px-6 py-3.5 flex group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Link",
        }
      }
    />
  </span>
</a>
`;

exports[`Buttons Matches snapshot for Secondary 1`] = `
<a
  className="border-vital-primary-interactive text-vital-primary-interactive hover:opacity-70 active:text-vital-primary-interactive-active border-2 hover:opacity-80 leading-tight uppercase focus:outline-none focus:ring-0 rounded transition duration-150 ease-in-out px-6 py-3.5 flex group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Link",
        }
      }
    />
  </span>
</a>
`;

exports[`Buttons Matches snapshot for SecondarySelected 1`] = `
<a
  className="border-vital-secondary-button-selected text-vital-secondary-button-selected border-2 hover:opacity-80 leading-tight uppercase focus:outline-none focus:ring-0 rounded transition duration-150 ease-in-out px-6 py-3.5 flex group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Link",
        }
      }
    />
  </span>
</a>
`;

exports[`Buttons Matches snapshot for Tertiary 1`] = `
<a
  className="text-vital-primary-interactive hover:opacity-70 active:text-vital-primary-interactive-active leading-tight uppercase focus:outline-none focus:ring-0 rounded-lg transition duration-400 ease-in-out px-6 py-4 flex group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Link",
        }
      }
    />
  </span>
</a>
`;

exports[`Buttons Matches snapshot for TertiarySelected 1`] = `
<a
  className="text-vital-secondary-button-selected leading-tight uppercase focus:outline-none focus:ring-0 rounded-lg transition duration-400 ease-in-out px-6 py-4 flex group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Link",
        }
      }
    />
  </span>
</a>
`;

exports[`Buttons Matches snapshot for WhereToBuy 1`] = `
<a
  className="rounded text-m-base lg:text-base uppercase font-bold text-vital-primary-footer-copy bg-vital-primary-interactive mx-auto p-3 w-full flex justify-center items-center max-w-64 h-12 lg:w-full"
  href="/where-to-buy"
>
  <svg
    className="w-6 h-6 mr-3 xl:mr-0 2xl:mr-3"
    height="25"
    viewBox="0 0 27.27 25"
    width="27.27"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="CART ICON"
      id="CART_ICON"
      transform="translate(1 1)"
    >
      <path
        d="M-2020.75,2766.846h4.712l3.426,15.993h14.314l2.818-11h-19.487"
        data-name="Path 5710"
        fill="none"
        id="Path_5710"
        stroke="#fff"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="2"
        transform="translate(2020.75 -2766.846)"
      />
      <circle
        cx="2.404"
        cy="2.404"
        data-name="Ellipse 1822"
        fill="#fff"
        id="Ellipse_1822"
        r="2.404"
        transform="translate(8.019 19.192)"
      />
      <circle
        cx="2.404"
        cy="2.404"
        data-name="Ellipse 1823"
        fill="#fff"
        id="Ellipse_1823"
        r="2.404"
        transform="translate(17.657 19.192)"
      />
    </g>
  </svg>
  <span
    className="leading xl:hidden 2xl:block"
  >
    Where to Buy
  </span>
</a>
`;

exports[`Buttons Matches snapshot for WhereToBuyWithoutIcon 1`] = `
<a
  className="rounded text-m-base lg:text-base uppercase font-bold text-vital-primary-footer-copy bg-vital-primary-interactive mx-auto p-3 w-full flex justify-center items-center max-w-64 h-12 lg:w-full"
  href="/where-to-buy"
>
  <span
    className="leading xl:hidden 2xl:block"
  >
    Where to Buy
  </span>
</a>
`;

exports[`Buttons Matches snapshot for WithArrow 1`] = `
<a
  className="pl-12 pr-6 py-3.5 group flex-row-reverse"
>
  <span
    className="w-6 h-2 vital-arrow group-hover:text-current text-transparent pr-1 inline-block"
  />
  <span />
</a>
`;

exports[`Buttons Matches snapshot for WithBigButton 1`] = `
<a
  className="hover:opacity-80 leading-tight uppercase focus:outline-none focus:ring-0 rounded transition duration-150 ease-in-out px-12 py-6 flex group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Link",
        }
      }
    />
  </span>
</a>
`;

exports[`Buttons Matches snapshot for WithDisabled 1`] = `
<button
  className="opacity-50 hover:opacity-50"
  disabled="true"
>
  <span />
</button>
`;

exports[`Buttons Matches snapshot for WithPrimary 1`] = `
<a
  className="rounded hover:bg-vital-primary-interactive transition-colors duration-400 text-m-base lg:text-base uppercase font-bold text-white bg-vital-primary-brand px-6 py-3.5 flex group justify-center"
  href="#"
  role="button"
>
  <span
    data-no-hydrate={true}
    id="123b58b4a9e4407d42b2606d912b6deb"
    style={
      Object {
        "display": "contents",
      }
    }
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Link",
        }
      }
    />
  </span>
</a>
`;
